apps/client/src/app/pages/food-picker/food-picker/food-picker.component.ts

```diff
@@ -0,0 +1,90 @@
+import { Component } from '@angular/core';
+import { BehaviorSubject, combineLatest, Observable, Subject } from 'rxjs';
+import { debounceTime, map, switchMap, tap, filter } from 'rxjs/operators';
+import * as _ from 'lodash';
+import { stats } from '../../../core/data/sources/stats';
+import { TeamcraftComponent } from '../../../core/component/teamcraft-component';
+import { SearchIndex, XivapiSearchFilter, XivapiService } from '@xivapi/angular-client';
+import { LazyDataService } from '../../../core/data/lazy-data.service';
+
+@Component({
+  selector: 'app-food-picker',
+  templateUrl: './food-picker.component.html',
+  styleUrls: ['./food-picker.component.less'],
+})
+export class FoodPickerComponent extends TeamcraftComponent {
+
+  public availableStats = stats;
+
+  public stats$ = new BehaviorSubject<any[]>([null]);
+  
+  public results$: Observable<any[]>;
+
+  public loading = false;
+
+  constructor(private xivapi: XivapiService, private lazyData: LazyDataService) {
+    super();
+
+    this.results$ = this.stats$.pipe(
+      filter(stats => {
+        return stats.length > 0 && stats.every(s => s !== null);
```

Comment:
The `stats` property is already in the scope because of the class field, this creates a shadowed variable which is forbidden by the linter rules.

You might want to include tslint to your IDE in order to apply these rules automatically.

To check linter, use `npm run lint`.

Reply:
Oh yeah didn't noticed, I'll fix it